src/icalendar/prop/__init__.py

```diff
@@ -272,6 +372,11 @@ def to_ical(self):
     def from_ical(cls, ical):
         return cls(ical)
 
+    @property
+    def ical_value(self):
+        """the mailto: part of the address"""
```

Comment:
This is broken. https://icalendar--979.org.readthedocs.build/en/979/reference/api/icalendar.prop.html#icalendar.prop.vCalAddress.ical_value

```suggestion
        """The ``mailto:`` part of the address."""
```